http/jakarta-rest-reactive/src/test/java/io/quarkus/ts/http/jakartarest/reactive/OpenShiftExecutionModelIT.java

```diff
@@ -3,11 +3,13 @@
 import java.util.function.Predicate;
 
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import io.quarkus.test.scenarios.OpenShiftScenario;
 
 @OpenShiftScenario
+@Disabled("https://issues.redhat.com/browse/QQE-465")
```

Comment:
I like the change, but please keep in mind that QQE is not a public tracker.

Can you refer to a github issue instead, please?

Reply:
OK changed.